packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1900,6 +1900,10 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             Array.isArray(pkg) ? pkg : [pkg], id, isRoot, props).realize();
         if (isRoot) {
             fluidDataStore.bindToContext();
+            this.logger.sendTelemetryEvent({
```

Comment:
In general, I do not think telemetry like that (especially if it's not an error) is helpful. We should assume that we can see telemetry only from small set of our customers. For example, I think onenote does not send data to our DB. Same for any 3rd party.

Telemetry like that might be good at nudging people to react if it brings attention of developers on FF upgrade - so issuing error events might have some impact.

But when it comes to end-to-end process, I think we should not rely on telemetry, we should rely on process itself - clearly setting SLAs, expectations, communication (breaking.md, deprecated APIs).

Reply:
I agree in general. This is to augment the process, provide the data (if available) about the _main_ customers that already use the API to monitor the decommission